<launch>
	<!--<env name="ROS_MASTER_URI" value ="http://rov:11311/"></env>-->
	<node name="controls_node" pkg="jaws_controls" type="controls_node">
	<param name="stbd_thrust_multiplier" value="1"/>
	<param name="port_thrust_multiplier" value="1"/>
	<param name="controls_refresh_rate" value="10"/>
	</node>
	<node name = "joy_node" pkg="joy" type="joy_node"></node>
</launch>
